source/modules/misc/arguments.py

```diff
@@ -85,6 +89,10 @@ def validate_arguments(arguments: Namespace) -> dict:
                   f" Available commands are: {sub_string}.{PRINT_END}")
             sys.exit(57)
```

Comment:
Exit only from main and instead return None/integer?

Reply:
I don't think we should change that approach before v1